pkg/reconciler/pipelinerun/resources/pipelinerunstate.go

```diff
@@ -542,6 +542,29 @@ func (facts *PipelineRunFacts) GetPipelineTaskStatus() map[string]string {
 	return tStatus
 }
 
+func (facts *PipelineRunFacts) GetPipelineFinalTaskStatus() map[string]string {
```

Comment:
Can you add a doc string for this function?

Reply:
Thanks!!